package.json

```
@@ -9,9 +9,10 @@
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "nest start",
-    "start:dev": "nest start --watch",
+    "start:dev": "SET NODE_ENV=development&&nest start --watch",
```

Can we put spaces to be a bit clearer:
```suggestion
    "start:dev": "set NODE_ENV=development && nest start --watch",
```